### ISSUE TITLE
Could you explain what to do if GuardDuty is enabled? 

### ISSUE THREAD

**Issue by jarrettj** [aws-samples/amazon-guardduty-for-aws-organizations-with-terraform#5]
Hi,

Good day.

Under Limitations you list the following:

GuardDuty must not already be enabled in any of the accounts, in any of the target Regions.

Does this mean we have to disable the service first before proceeding? Also, what happens if it is enabled? Thanks.

Regards,
Jarrett

**Comment by aarthkan** [comment]
Hi,
This is a limitation that I observed when testing the code for the pattern.
When trying to run this pattern, if the service is already enabled, Terraform fails with an error. I have observed the same when deploying the service using CloudFormation templates as well. Hence the recommendation to make sure that the service is disabled before testing this pattern. 
If you have enabled the service using the member-invite based method, I have provided set of steps in the GitHub repo to disable the setup methodically, before deploying the pattern.
Hope this helps !

Regards,
Aarthi
